src/web/mjs/connectors/BilibiliManhua.mjs

```diff
@@ -23,26 +23,83 @@ export default class BilibiliManhua extends Connector {
         };
         //default config doesnt need language switch, its fixed to CN
         this.config = {
-            quality:  {
-                label: 'Preferred format',
-                description: 'format of images\nwebp (low)\njpg (medium)\npng (high))',
-                input: 'select',
-                options: [
-                    { value: 'webp', name: 'webp' },
-                    { value: 'jpg', name: 'jpg' },
-                    { value: 'png', name: 'png' },
+            format: {
+                label:'Preferred format',
+                description:'Format of images\nwebp(low)\n jpg(medium)\n png(high))',
+                input:'select',
+                options:[
+                    { value:'webp', name:'webp' },
+                    { value:'jpg', name:'jpg' },
+                    { value:'png', name:'png' },
                 ],
                 value: 'png'
             },
             language: {
                 label: 'Language Settings',
-                description: 'Choose the language to use. This will affect available manga lists',
+                description: 'Choose the language to use. This will affect available manga in list',
                 input: 'disabled',
-                value : 'cn'
+                value: 'cn'
+            },
+            picquality: {
+                label: 'Quality Settings',
+                description: 'Choose the prefered quality',
+                input: 'select',
+                options: [
+                    { value: 'veryhigh', name:'VeryHigh' },
+                    { value: 'good', name:'Good' },
+                    { value: 'normal', name:'Normal' },
+                    { value: 'poor', name:'Poor' },
+                    { value: 'verypoor', name:'VeryPoor' },
+                ],
+                value: 'good'
+            },
+            forcepicturesize: {
+                label: 'Force max quality (Experimental)',
+                description: 'Force server to send pictures with maxsize. Override "quality settings"',
+                input: 'checkbox',
+                value: false
             }
         };
     }
 
+    getImageSizeByQuality(width) {
```

Comment:
Business logic is hard to understand with less expressive variable names such as `V`, `J`, `o.w`, `o.q`, `n`, `w`
Consider using more expressive local variable names, or if the method was extracted from website add a related comment.

Reply:
Yup it was extracted from website, but it doesnt mean i cant use better variables.